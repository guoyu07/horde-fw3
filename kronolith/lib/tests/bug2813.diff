001- array(7) {
001+ array(6) {
014-   ["_supportedSpecs"]=>
014+ }
015-   string(21) "%CdDeHImMnRStTyYbBpxX"
015+ array(6) {
016- }
016+   ["year"]=>
017- array(7) {
017+   int(2006)
018-   ["year"]=>
018+   ["month"]=>
019-   int(2006)
019+   int(6)
020-   ["month"]=>
020+   ["mday"]=>
021-   int(6)
021+   int(17)
022-   ["mday"]=>
022+   ["hour"]=>
023-   int(17)
023+   int(18)
024-   ["hour"]=>
024+   ["min"]=>
025-   int(18)
025+   int(0)
026-   ["min"]=>
026+   ["sec"]=>
028-   ["sec"]=>
028+ }
029-   int(0)
029+ array(1) {
030-   ["_supportedSpecs"]=>
030+   [0]=>
031-   string(21) "%CdDeHImMnRStTyYbBpxX"
031+   bool(false)
033- array(1) {
034-   [0]=>
035-   bool(false)
036- }