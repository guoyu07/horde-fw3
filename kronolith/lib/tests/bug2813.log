
---- EXPECTED OUTPUT
array(7) {
  ["year"]=>
  int(2006)
  ["month"]=>
  int(6)
  ["mday"]=>
  int(16)
  ["hour"]=>
  int(18)
  ["min"]=>
  int(0)
  ["sec"]=>
  int(0)
  ["_supportedSpecs"]=>
  string(21) "%CdDeHImMnRStTyYbBpxX"
}
array(7) {
  ["year"]=>
  int(2006)
  ["month"]=>
  int(6)
  ["mday"]=>
  int(17)
  ["hour"]=>
  int(18)
  ["min"]=>
  int(0)
  ["sec"]=>
  int(0)
  ["_supportedSpecs"]=>
  string(21) "%CdDeHImMnRStTyYbBpxX"
}
array(1) {
  [0]=>
  bool(false)
}
---- ACTUAL OUTPUT
array(6) {
  ["year"]=>
  int(2006)
  ["month"]=>
  int(6)
  ["mday"]=>
  int(16)
  ["hour"]=>
  int(18)
  ["min"]=>
  int(0)
  ["sec"]=>
  int(0)
}
array(6) {
  ["year"]=>
  int(2006)
  ["month"]=>
  int(6)
  ["mday"]=>
  int(17)
  ["hour"]=>
  int(18)
  ["min"]=>
  int(0)
  ["sec"]=>
  int(0)
}
array(1) {
  [0]=>
  bool(false)
}
---- FAILED
