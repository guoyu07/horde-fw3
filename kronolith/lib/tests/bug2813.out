array(6) {
  ["year"]=>
  int(2006)
  ["month"]=>
  int(6)
  ["mday"]=>
  int(16)
  ["hour"]=>
  int(18)
  ["min"]=>
  int(0)
  ["sec"]=>
  int(0)
}
array(6) {
  ["year"]=>
  int(2006)
  ["month"]=>
  int(6)
  ["mday"]=>
  int(17)
  ["hour"]=>
  int(18)
  ["min"]=>
  int(0)
  ["sec"]=>
  int(0)
}
array(1) {
  [0]=>
  bool(false)
}